File: app/src/app/core/services/api/models/naslov.ts
import { Id } from './id'
import { Oseba } from './oseba'
export interface Naslov {
	'_id': Id<Naslov>
	dodatno: string
	drzava: string
	mesto: string
	oseba_id: Id<Oseba>
	ulica: string
	zip: string
}

File: api/core/src/main/kotlin/core/domain/Naslov.kt
package core.domain

import core.base.Id
import kotlinx.serialization.Contextual
import kotlinx.serialization.Serializable

@Serializable
data class Naslov(
    override var _id: Id<Naslov> = Id(),
    var oseba_id: Id<Oseba>,
    @Contextual val drzava: core.base.Encrypted,
    @Contextual val mesto: core.base.Encrypted,
    @Contextual val zip: core.base.Encrypted,
    @Contextual val ulica: core.base.Encrypted,
    @Contextual val dodatno: core.base.Encrypted
) : Entiteta<Naslov>
